### DISCUSSION TITLE
匯入項目後,量算工具组件按鈕位置變了

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#353] [Q&A]
您好,
單獨使用您的量算工具組件沒問題
但是匯入項目後,
VcMeasurements按鍵佈局跑掉
VcStatusBar 佈局也跑掉
請問可以修改CSS那個設置?
![2023-02-16_085738](https://user-images.githubusercontent.com/5283322/219234167-425bf83f-09a0-4809-9a75-30b298954085.png)

**Comment by zouyaoji** [comment]
请提供复现代码看看。

**Comment by stan88** [comment]
您好,
項目內是匯入這個CSS後,VcMeasurements 與 VcStatusBar 佈局跑掉

[AspStudio.min.zip](https://github.com/zouyaoji/vue-cesium/files/10772927/AspStudio.min.zip)

**Reply by zouyaoji** [reply]
那应该是css污染了。你在引入 AspStudio只后引入vue-cesium的css。  如果还不行就只有自己看一下是哪个css影响了，自己重置一下。

**Comment by stan88** [comment]
能否提示下VcMeasurements 與 VcStatusBar具體用到那些CSS,
找了二天...還是沒試著...

**Reply by zouyaoji** [reply]
在浏览器里，检查元素自己看哦。
![image](https://user-images.githubusercontent.com/24430116/219832815-5541a3aa-2dbf-4e81-b22e-c1d23bad7feb.png)
